src/auro-select.js

```diff
@@ -69,6 +72,10 @@ class AuroSelect extends LitElement {
         type: Boolean,
         reflect: true
       },
+      checkmark: {
+        type: Boolean,
+        reflect: true
+      },
```

Comment:
To remove the necessity of a MAJOR release for this update that is inconsistent with how the other `noCheckmark` attributes are working with similar UIs, we should reverse this logic to use `noCheckmark` for a MINOR release. 

There is work to be addressed in the upcoming months regarding forms and consistency between them where we can revisit this feature across all similar UI elements and if decided to have major changes we will release MAJOR versions of all form elements at the same time.